contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
The kubernetes repo is special, because it spans most/all SIGs and has very high issue volume. Smaller, more-focused repos don't have the same challenges. And, issue labels and bot automation don't yet apply to all repos, since they have to be set up for each repo individually.